cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -61,14 +61,13 @@ def _include_file(self, root_parts, f):
         return True
 
     def _get_files_to_package(self, path):
-        files_to_package = []
+        # files_to_package = []
```

Comment:
@spelak-salesforce can you remove this instead of commenting it out?

Reply:
It's removed in a later commit.  Sorry it was snuck in.  I commented out to see if the test still passed before full removal.

Reply:
oh yeah, sorry. I should look at the full diff rather than individual commits for reviewing.

Reply:
No, it's still there.  removing...

Reply:
Done.